binary/test/Test/Cardano/Binary/Helpers/GoldenRoundTrip.hs

```diff
@@ -65,6 +76,13 @@ failHexDumpDiff x y = case hexDumpDiff x y of
     ["━━━ Not Equal ━━━", showPretty x, showPretty y]
   Just dif -> withFrozenCallStack $ failWith Nothing $ renderHexDumpDiff dif
 
+-- | Check that the 'encode' and 'decode' function of the 'Bi' instances work as
+-- expected w.r.t. the give reference data, this is, given a value @x::a@, and
+-- a file path @fp@:
+--
+-- - The encoded data should coincide with the contents of the @fp@.
+-- - Decoding @fp@ should give as a result @x@
+--
 goldenTestBi :: (Bi a, Eq a, Show a, HasCallStack) => a -> FilePath -> Property
 goldenTestBi x path = withFrozenCallStack $ do
```

Comment:
I think we should define this one in terms of the other one!

Reply:
Done!